packages/block-library/src/pullquote/index.js

```diff
@@ -47,51 +64,89 @@ export const settings = {
 
 	attributes: blockAttributes,
 
+	styles: [
+		{ name: 'default', label: __( 'Regular' ), isDefault: true },
+		{ name: STYLIZED_STYLE_NAME, label: __( 'Stylized' ) },
+	],
+
 	supports: {
 		align: true,
 	},
 
-	edit( { attributes, setAttributes, isSelected, className } ) {
-		const { value, citation } = attributes;
+	edit: withColors( 'backgroundColor' )(
+		( { attributes, backgroundColor, setAttributes, isSelected, className, setBackgroundColor } ) => {
+			const { value, citation } = attributes;
 
-		return (
-			<figure className={ className }>
-				<blockquote>
-					<RichText
-						multiline="p"
-						value={ toRichTextValue( value ) }
-						onChange={
-							( nextValue ) => setAttributes( {
-								value: fromRichTextValue( nextValue ),
-							} )
-						}
-						/* translators: the text of the quotation */
-						placeholder={ __( 'Write quote…' ) }
-						wrapperClassName="block-library-pullquote__content"
-					/>
-					{ ( ! RichText.isEmpty( citation ) || isSelected ) && (
-						<RichText
-							tagName="cite"
-							value={ citation }
-							/* translators: the individual or entity quoted */
-							placeholder={ __( 'Write citation…' ) }
-							onChange={
-								( nextCitation ) => setAttributes( {
-									citation: nextCitation,
-								} )
-							}
+			let colorSettings = null;
+			const isStylizedStyleVariation = includes( className, STYLIZED_CLASS );
+			if ( isStylizedStyleVariation ) {
```

Comment:
I was thinking we could actually try calling this attribute just "Main Color", and use it in the default version for the thick borders.